sql/moz-fx-data-marketing-prod/ga_derived/www_site_hits_v2/metadata.yaml

```
@@ -0,0 +1,22 @@
+friendly_name: WWW Site Hits V2
+description: |-
+  GA4 normalized events for www.mozilla.org; events triggered at the same time are given the same hit number.
+owners:
+- kwindau@mozilla.com
+labels:
+  incremental: true
+  owner1: kwindau@mozilla.com
+scheduling:
+  dag_name: bqetl_google_analytics_derived_ga4
+bigquery:
+  time_partitioning:
+    type: day
+    field: date
+    require_partition_filter: true
+    expiration_days: null
+  clustering:
+    fields:
+    - country
+    - language
```

Looking at the data, it seems like the event name is a good candidate for clustering: limited values and likely common querying.

```suggestion
    - language
    - event_name
```